src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $background-color;
+  width: 200px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__info {
+    margin: 36px 16px 16px;
+    color: #060B35;
+
+    &-title {
+      width: 166px;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 18px;
+      margin: 0 0 4px;
+    }
+
+    &-text {
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+      color: $text-color;
+      margin: 0 0 16px;
+    }
+
+    &-rate {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin: 0 0 24px;
+    }
+
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      margin: 0 0 16px;
+    }
+
+    &-button {
+      cursor: pointer;
+      width: 100%;
+      padding: 12px 0;
+      border-radius: 5px;
+      background-color: $button-color;
+      border: none;
+      color: #fff;
```

Comment:
Consider using a variable here

Reply:
Yes, but I can't add $background-color: #fff; for text color , becouse this variable created for background. Also in my project only one color: #fff;